docs/monitoring.md

```diff
@@ -0,0 +1,294 @@
+           "config": {
+                "name": "resnet",
+                "base_path": "/workspace/resnet-50-tf",
+                "layout": "NHWC:NCHW",
+                "shape": "(1,224,224,3)"
+           }
+        }
+    ],
+    "monitoring":
+        {
+            "metrics":
+            {
+                "enable" : true,
+                "metrics_list": 
+                    [ "ovms_requests_success",
+                    "ovms_requests_fail",
+                    "ovms_inference_time_us",
+                    "ovms_wait_for_infer_req_time_us",
+                    "ovms_request_time_us",
+                    "ovms_current_requests",
+                    "ovms_infer_req_active",
+                    "ovms_streams",
+                    "ovms_infer_req_queue_size"]
+            }
+        }
+    }
+    ```
+
+## Example response from metrics endpoint
+
+To use data from metrics endpoint you can use the curl command:
+```bash
+    curl http://localhost:8000/metrics
+```
+
+The example output looks like this:
+```bash
+    # HELP ovms_requests_success Number of successful requests to a model or a DAG.
+    # TYPE ovms_requests_success counter
+    ovms_requests_success{api="KServe",interface="REST",method="ModelMetadata",name="resnet",version="1"} 0
+    ovms_requests_success{api="KServe",interface="gRPC",method="ModelReady",name="resnet"} 0
+    ovms_requests_success{api="KServe",interface="gRPC",method="ModelInfer",name="resnet",version="1"} 0
+    ovms_requests_success{api="KServe",interface="REST",method="ModelInfer",name="resnet",version="1"} 0
+    ovms_requests_success{api="KServe",interface="gRPC",method="ModelMetadata",name="resnet",version="1"} 0
+    ovms_requests_success{api="TensorFlowServing",interface="REST",method="Predict",name="resnet",version="1"} 0
+    ovms_requests_success{api="TensorFlowServing",interface="gRPC",method="GetModelStatus",name="resnet"} 0
+    ovms_requests_success{api="KServe",interface="REST",method="ModelReady",name="resnet"} 0
+    ovms_requests_success{api="TensorFlowServing",interface="REST",method="GetModelStatus",name="resnet"} 0
+    ovms_requests_success{api="TensorFlowServing",interface="gRPC",method="GetModelMetadata",name="resnet",version="1"} 0
+    ovms_requests_success{api="TensorFlowServing",interface="REST",method="GetModelMetadata",name="resnet",version="1"} 0
+    ovms_requests_success{api="TensorFlowServing",interface="gRPC",method="Predict",name="resnet",version="1"} 0
+```
+
+## Differences between metrics against model or a DAG
+
+For DAG pipeline execution there are six metrics available:
+
+DAG METRICS
+
+   ```bash
+         counter     ovms_requests_success               Number of successful requests to a model or a DAG.
+         counter     ovms_requests_fail                  Number of failed requests to a model or a DAG.
+         histogram   ovms_request_time_us                Processing time of requests to a model or a DAG.
+   ```
+
+## Example graphics from Graphana
```

Comment:
We can add this section when the graphs are ready to show

Reply:
Ok.